about.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Page</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c90d8a2f33.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+  <header>
+    <nav class="social-head">
+      <ul class="nav-list">
+        <li><a href="#"><i class="fa-brands fa-facebook-square"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-twitter-square"></i></a></li>
+        <li><a href="#">Logout</a></li>
+        <li><a href="#">School Page</a></li>
+        <li><a class="engo" href="#">English</a></li>
+      </ul>
+    </nav>
+
+
+    <nav class="navbar">
+      <div class="ham" id="ham">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <a href="index.html" class="brand"><img src="./imgs/uonlogo.png" alt="uon logo"></a>
+      <ul class="navmenu" id="navmenu">
+          <li class="items item1"><a href="about.html" class="about-link" id="about">About</a></li>
```

Comment:
- [ ] [OPTIONAL] You don't have a link to the home page from the about page beside the logo which is not very obvious to use, please consider adding a link to the home page. you can use the about link for that e.g. 
   ```
     <li class="items item1">
          <a href="index.html" class="about-link" id="about">Home</a>
      </li>
  ```